consensus/consortium/v2/consortium.go

```diff
@@ -755,15 +806,17 @@ func (c *Consortium) Seal(chain consensus.ChainHeaderReader, block *types.Block,
 	}
 
 	// Sweet, the protocol permits us to sign the block, wait for our time
-	delay := time.Unix(int64(header.Time), 0).Sub(time.Now()) // nolint: gosimple
-	if header.Difficulty.Cmp(diffInTurn) != 0 {
-		// It's not our turn explicitly to sign, delay it a bit
-		wiggle := time.Duration(len(snap.Validators)/2+1) * wiggleTime
-		delay += time.Duration(rand.Int63n(int64(wiggle))) + wiggleTime // delay for 0.5s more
-
-		log.Trace("Out-of-turn signing requested", "wiggle", common.PrettyDuration(wiggle))
+	// After the Buba hardfork, the delay is included in header time already
+	delay := time.Until(time.Unix(int64(header.Time), 0))
+	if !c.chainConfig.IsBuba(block.Number()) {
```

Comment:
should it be?
- before Buba: keep the old behavior
- after Buba: new behavior

Here I see you changed the way calculating delay time even when chains don't meet Buba! Can we keep it backward compatible?

Reply:
Before Buba, the header time does not include delay, after Buba, the delay is included into header time for verification. So the point of this code is if not Buba, calculate the additional delay, otherwise, because the delay is included in header time, just do nothing more and wait until the timestamp in header time.